### PR TITLE
Tweaks to go-lxc usage

### PR DESCRIPTION
**Pull request by stgraber**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -25,6 +25,7 @@ require (
 	github.com/juju/gomaasapi v0.0.0-20200602032615-aa561369c767
 	github.com/juju/persistent-cookiejar v0.0.0-20171026135701-d5e5a8405ef9
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
+	github.com/lxc/go-lxc v0.0.0-20220627182551-ad3d9f7cb822
```
Comment (tomponline) [inline go.mod:28]: Is it worth tagging go-lxc v2 for this rather than using a commit?

**Review by tomponline — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (stgraber) [reply go.mod:28]: No, the lack of any tags means that every time we run `make update` it picks the latest commit, this is fine.

Comment (tomponline) [reply go.mod:28]: Ok, out of interest what makes it use the v2 branch?

**Comment by brauner** [comment]
Thanks you!

**Pull request merged**